app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name, age) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list) -> list:
+    person_instances = []
+
+    for person in people_list:
+        person_instance = Person(person["name"], person["age"])
+        person_instances.append(person_instance)
+
+    for person in people_list:
+        if "wife" in person and person["wife"]:
+            wife_name = person["wife"]
+            if wife_name in Person.people:
+                Person.people[person["name"]].wife = Person.people[wife_name]
+            else:
+                print(f"Warning: Wife {wife_name} not found for {person["name"]}")
```

Comment:
The string formatting here is incorrect. You should use single quotes around the dictionary key to avoid syntax errors. Change `person["name"]` to `person['name']`.